lib/able_git_hooks/generators/install_generator.rb

```diff
@@ -22,6 +25,33 @@ def install_rubocop_hooks
       def copy_rubocop_defaults
         template "rubocop.yml", "config/rubocop.yml"
       end
+
+      def install_eslint_hooks
+        copy_file "eslint-check.js", "hooks/pre-commit/eslint-check.js"
+        chmod "hooks/pre-commit/eslint-check.js", 0755
+      end
+
+      def copy_eslint_defaults
+        copy_file "eslintrc.json", ".eslintrc.json"
+      end
+      
+      def copy_prettier_defaults
+        copy_file "prettierrc", ".prettierrc"
+      end
+
+      def install_stylelint_hooks
+        puts "options: ", options
+        if options['with_stylelint']
```

Comment:
Rubocop will complain about ' vs "

Reply:
/me nods

I've included the rubocop.yml and enabled rubocop in my editor to prevent this style issues for occurring in the future. There where also a couple of trailing whitespace in the buffer